recipes/isl/build.sh

```diff
@@ -0,0 +1,10 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
+make
+
+if [ "$(uname)" = 'Darwin' ]
+then
+    export DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib"
+fi
+make check
+
```

Comment:
I suppose we should unset `DYLD_FALLBACK_LIBRARY_PATH` after testing, but I can't see it hurting the install step.